sdk/storage/Azure.Storage.Blobs/src/BlockBlobClient.cs

```diff
@@ -548,14 +548,14 @@ protected sealed override BlobBaseClient WithSnapshotImpl(string snapshot)
         /// notifications that the operation should be cancelled.
         /// </param>
         /// <returns>
-        /// A <see cref="Response{BlobContentInfo}"/> describing the
-        /// state of the updated block blob.
+        /// A <see cref="Response{BlockInfo}"/> describing the
```

Comment:
I imagine there was a conversation about this type name `BlockInfo`? usually type names are complete and not abbreviations. Also, `Info` doesn't help telling the user what the type is or what type of content it has...

Reply:
We've been using `Info` all over the place for our Response types.  This is one of those "okay for Preview" gray areas the Arch Board wants to lock down before the next release.